docs/lightning/lnd-alias.md

```diff
@@ -2,16 +2,8 @@
 
 Every Lightning node has a name that can be configured. By default, each myNode device uses the name "mynodebtc.com [myNode]". TO change this name first go to the LND page in teh GUI and click the gear icon in the Alias box.
 
-<center>
-  <figure>
-    <img src="/images/lightning/setting-lnd-alias-1.png" alt="myNode logo" style="width: 300px">
-  </figure>
-</center>
+![](/images/lightning/alias-1.png)
 
 After clicking on the gear icon, you will be presented with a popup that allows you to change your node's alias. Enter the new alias, your password, and click Save. This will reboot you device and it will come with the new alias!
```

Comment:
```suggestion
After clicking on the gear icon, you will be presented with a popup that allows you to change your node's alias. Enter the new alias, your password, and click Save. This will reboot your device and it will come with the new alias!
```